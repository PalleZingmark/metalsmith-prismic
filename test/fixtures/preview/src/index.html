---
template: index.hbt
prismic:
  cupcake:
    query: '[[:d = at(document.tags, ["Cupcake"])] [:d = any(document.type, ["product"])]]'
    orderings: '[my.product.name]'
---
