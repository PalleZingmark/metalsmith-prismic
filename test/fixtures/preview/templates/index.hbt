<html>
<body>
<ul>
{{#each prismic.cupcake.results}}
    <li>
        {{{data.name.html}}}
        {{{data.description.html}}}
    </li>
{{/each}}
</ul>
</body>
</html>
