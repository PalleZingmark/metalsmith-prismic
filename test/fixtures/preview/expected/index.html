<html>
<body>
<ul>
    <li>
        <h1>Golden Gate Cupcake</h1>
        <p>The launch of <a href="/store/UlfoxUnM0wkXYXbq/san-francisco-pier-39">our brand new San Francisco shop</a> would not have been complete without a proper celebration in taste! An occasion to see the berries on its top (as red and sunny as the Golden Gate Bridge) mingle with the soft fruit in its cake (as natural as San Francisco's organic food). Also, an occasion to celebrate high-tech, because it is the occasion to see the encounter of... apples and blackberries! A state of the art pastry, if you will...</p>
    </li>
    <li>
        <h1>Hot Berry Cupcake</h1>
        <p>You must remember your auntie Judith's garden, and the wonderful berries she was always picking up for dessert? Well, your inner child's heart is going to have to meet the enjoyable roughness of adulthood: when strawberry, cranberry and rapsberry meet tabasco, it's the powerful taste mix that wins the game.</p>
    </li>
    <li>
        <h1>Soft Lemon Cupcake</h1>
        <p>Nobody blames lemon for its bitterness. And there are times when, more than ever, nobody should! How could bitterness be to blame, while it is so well enclosed in a sweetening-chocolatey setting? And since we don't doubt you are the fine type, you get to have a choice between a softer version (topped with the sweetest bit of white chocolate you could find), or a tougher version (topped with a provocative bitter lemon jelly). Sweet white chocolate + lemon bitterness = the magic power of an unexpected encounter.</p>
    </li>
    <li>
        <h1>Speculoos Cupcake</h1>
        <p>It was yet another travel that <em>Les Bonnes Choses</em> had to offer you. Take a sweet trip to the wonderful Brussels, and as you enjoy a typically Belgian coffee with waffles, let yourself bite into the purest expression of the typical Speculoos taste, concentrated into a single cupcake.</p>
    </li>
    <li>
        <h1>Triple Chocolate Cupcake</h1>
        <p>Do you know what's better than double-chocolate? Triple-chocolate! The chocolate cake at the bottom supports the chocolate cream, which supports tiny, soft minced chocolate ; and together, they support an overwhelming taste of chocolate in your mouth.</p>
    </li>
</ul>
</body>
</html>
